utils/arp-whisper/Makefile

```
@@ -0,0 +1,53 @@
+# SPDX-License-Identifier: GPL-3.0-only
+#
+# Copyright (C) 2023 Facundo Acevedo
+
+include $(TOPDIR)/rules.mk
+
+PKG_NAME:=arp-whisper
+PKG_VERSION:=0.1.2
+PKG_RELEASE:=1
+
+PKG_SOURCE:=arp-whisper-$(PKG_VERSION).tar.gz
+PKG_SOURCE_URL:=https://codeload.github.com/FacundoAcevedo/arp-whisper/tar.gz/v$(PKG_VERSION)?
+PKG_HASH:=13b393c64505d62b5abf7ace244eed0ce62f4b1562a688ddb5651bd42cc7c305
+
+PKG_MAINTAINER:=Facundo Acevedo <facevedo@disroot.org>
+PKG_LICENSE:=GPL-3.0-or-later
+PKG_LICENSE_FILES:=COPYING
+
+PKG_BUILD_DEPENDS:=rust/host
+
+include $(INCLUDE_DIR)/package.mk
+include ../../lang/rust/rust-package.mk
+
+define Package/arp-whisper
+	SECTION:=utils
+	CATEGORY:=Utilities
+	TITLE:=Respond to ARP requests based on a configuration file of MAC-IP pairs.
+	DEPENDS:=$(RUST_ARCH_DEPENDS) 
+	URL:=https://github.com/FacundoAcevedo/arp-whisper
+endef
+
+define Package/arp-whisper/description
+	arp-whisper listens to ARP requests on a network interface and responds
+	to them based on a list of IP-MAC address mappings defined in a 
+	configuration file
+endef
```

These two has two spaces as indentation. It is quite odd, but it is what it is. The previous one was correct. The only section, which has tab is install section.

---

In what line? I'm using the mouse to highlight the text and they look like tabs. Also, I've rechecked the Makefile in my editor, I searched for Tabs, and all the indentations are done with them.

---

```suggestion
define Package/arp-whisper
  SECTION:=utils
  CATEGORY:=Utilities
  TITLE:=Respond to ARP requests based on a configuration file of MAC-IP pairs
  DEPENDS:=$(RUST_ARCH_DEPENDS) 
  URL:=https://github.com/FacundoAcevedo/arp-whisper
endef

define Package/arp-whisper/description
  arp-whisper listens to ARP requests on a network interface and responds
  to them based on a list of IP-MAC address mappings defined in a 
  configuration file
endef
```

It means you should use 2 spaces instead of tab for `Package/arp-whisper` and `Package/arp-whisper/description`